package.json

```diff
@@ -16,7 +16,7 @@
   },
   "main": "./index.js",
   "files": [
-    "dist/"
+    "build/"
```

Comment:
Need to update the constraints as well, I believe the dist file is referenced there

Reply:
here: https://github.com/MetaMask/test-dapp-multichain/blob/ff95a815068920f8c46aead84baccc8c5b315f2b/yarn.config.cjs#L255